.cargo/config

```diff
@@ -1,6 +1,7 @@
+# Target configuration for the example apps on the nRF52-DK
 [target.thumbv7em-none-eabi]
 rustflags = [
-  "-C", "link-arg=-Tlayout.ld",
-  "-C", "relocation-model=ropi-rwpi",
+  "-C", "link-arg=-Tnrf52_layout.ld",
```

Comment:
Oh I see, as-is if a crate includes libtock-rs using crates.io or a git = "..." dependency then it can't connect the linker scripts together.

I'll investigate cortex-m-quickstart next week and see how it solves this problem.

Reply:
As far as I understand, it includes the specific linker script using build.rs.

Reply:
I agree that would be nice, but I also feel it is out of scope for this PR. Unless I am missing something, the following facts are true both before and after this PR:

1. Clients depending on libtock-rs with a path = "" dependency can refer to the library's layout file directly
2. Clients depend on libtock-rs using a git = "" dependency or crates.io must have their own copy of the libtock-rs layout file.

In that respect, what you are asking for is an additional feature that belongs in its own PR.